tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: 2024 Q4 Hyperledger Aries
+parent: 2024
+grand_parent: Project Updates
+---
+
+## Project Health
+
+Work on [Hyperledger Aries] related projects continues at a healthy pace, albeit
+with some of that work now outside of the Aries itself. [ACA-Py] (formerly Aries
+Cloud Agent Python) achieved two significant milestones this quarter. Release
+1.0.0 was published, and the project moved to the [OpenWallet Foundation].
+Within [Hyperledger Aries], progress was made on all of the frameworks -- [Aries VCX],
+[Aries Framework Swift] and [Aries Framework Kotlin]. Other [Open Wallet
```

Comment:
```suggestion
[Aries Framework Swift] and [Aries Framework Kotlin]. Other [OpenWallet
```

Reply:
The title of the Foundation on the website has no space -- see: https://openwallet.foundation/

Reply:
Agree. Removing the space in my suggestion.

Reply:
Fixed